src/app/settings/action-triggers/components/action-triggers-list/action-triggers-list.component.ts

```diff
@@ -0,0 +1,241 @@
+    { key: `${ActionTriggerSortKeys.SHORTCUT}_${SortOrder.REVERSE_ALPHABETICAL}`, label: $localize`action-triggers-sort.shortcut-reverse-alphabetical.label` },
+  ]
+
+  public selectedActionTriggerOptions: SofieDroppdownOption[] = [
+    { key: UserActionsWithSelectedTriggers.DISABLE_SELECTION, label: $localize`action-triggers.disable-multi-selection.label`, disabled: false },
+    { key: UserActionsWithSelectedTriggers.TOGGLE_SELECT, label: $localize`global.select-all.label`, disabled: false },
+    { key: UserActionsWithSelectedTriggers.EXPORT, label: $localize`global.export-selected.label`, disabled: true },
+    { key: UserActionsWithSelectedTriggers.DELETE, label: $localize`global.delete-selected.label`, disabled: true },
+  ]
+  public selectMode: boolean = false
+  private readonly selectedActionTriggerIds: Set<string> = new Set()
+
+  constructor(
+    private readonly dialogService: DialogService,
+    private readonly fileDownloadService: HttpFileDownloadService,
+    private readonly actionTriggerService: ActionTriggerService
+  ) {}
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    const actionTriggerChange: SimpleChange | undefined = changes['actionTriggers']
+    if (actionTriggerChange) {
+      this.newSortSelect(this.sort)
+    }
+  }
+
+  get filteredActionsTriggers(): ActionTriggerWithActionInfo<KeyboardTriggerData>[] {
+    return this.actionTriggers.filter(
+      trigger =>
+        trigger.data.label.toLocaleLowerCase().includes(this.search.toLocaleLowerCase()) ||
+        trigger.data.keys.toString().toLocaleLowerCase().includes(this.search.toLocaleLowerCase()) ||
+        trigger.id === this.selectedActionTrigger?.id
+    )
+  }
+
+  public selectActionTrigger(actionTrigger: ActionTriggerWithActionInfo<KeyboardTriggerData> | undefined): void {
+    this.onActionTriggerOpen.emit(this.selectedActionTrigger?.id === actionTrigger?.id ? undefined : actionTrigger)
+  }
+
+  public setActionTriggerSelection(isSelected: boolean, id: string): void {
+    if (isSelected) {
+      this.selectedActionTriggerIds.add(id)
+    } else {
+      this.selectedActionTriggerIds.delete(id)
+    }
+    this.updateSelectedActionTriggersOptions()
+  }
+
+  public actionTriggerDelete(actionTriggerId: string): void {
+    this.dialogService.createConfirmDialog($localize`global.delete.label`, $localize`action-triggers.delete.confirmation`, 'Delete', () => this.deleteActionTriggerById(actionTriggerId))
+  }
+
+  private deleteActionTriggerById(actionTriggerId: string): void {
+    this.closeSelectedTabIfAreDeleted(actionTriggerId)
+    this.actionTriggerService.deleteActionTrigger(actionTriggerId).subscribe()
+  }
+
+  private closeSelectedTabIfAreDeleted(deletedActionId: string): void {
+    if (this.selectedActionTrigger?.id === deletedActionId) {
+      this.onActionTriggerOpen.emit(undefined)
+    }
+  }
```

Comment:
To move this functionality to the parent component, we need to make a new emitter specifically for it. Because otherwise at the moment we have no way to distinguish whether we will emit `this.onActionTriggerOpen.emit(undefined)` upon user click or from this logic that makes the closing automatically when this Action trigger is deleted

Reply:
Why do we need to discern the difference?

Reply:
Because currently we close the selection only when deleting an action trigger only if the deleted action trigger is the selected (open) one. Otherwise, I leave it open, because it doesn't make sense to me that if another one is deleted and there is one open, we should close it

Reply:
By having this component react to a local action for state management of an unknown component, the application will get in a wrong state if another browser is showing the page in the same state. Whether or not to display the action trigger form should be determined in the parent component; and it should be based on the action trigger state that is supplied as an input for the parent component.

Reply:
@KvelaGorrrrnio This logic will most likely disappear entirely after the redesign next sprint. So we decide to leave it that way for now.